less/home-page.less

```diff
@@ -24,6 +29,20 @@
       margin: 20px 2.5%;
       border-radius: 20px;
       height: 100%;
+
+      @media @phone {
+        width: 100%;
+      }
+    }
+
+    @media @phone {
+      .sky{
+        display: none;
+      }
+
+      .launch {
+        display: none;
+      }
```

Comment:
This is completely fine to do your media queries the normal css way.  But when you're using less, you can actually use nesting to put media queries nested inside your elements. that way you don't have to declare the element inside of the query.